umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/collection_progress.clj

```diff
@@ -0,0 +1,17 @@
+(ns cmr.umm-spec.xml-to-umm-mappings.collection-progress
+  "Functions for parsing UMM collection progress records out of XML documents of other formats."
+  (:require
+   [clojure.string :as string]
+   [cmr.common.xml.parse :refer :all]
+   [cmr.common.xml.simple-xpath :refer :all]
+   [cmr.umm-spec.util :as umm-spec-util]))
+
+(defn get-collection-progress
```

Comment:
1. I was thinking of making a shared directory and put the collection_progress.clj in there. 
   I'm not sure I like combining all the support functions for different fields together. At least it doesn't follow the
   style we currently are doing.
2. Good idea. It crossed my mind too about what if that "Not Provided" was changed. Done.